geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/RegionAdvisor.java

```diff
@@ -291,7 +291,7 @@ public Semaphore getVolunteeringSemaphore() {
       }
     }
     if (logger.isDebugEnabled()) {
-      logger.debug("This maps is sksk {} and size is {}", map, map.keySet().size());
+      logger.debug("This maps is sksk {} and size is {}", map, map.size());
```

Comment:
What is sksk? And `maps` should be `map`.

Reply:
Yeah, I don't have a clue. 🤷